.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  IMAGE_NAME: open_spiel
+
+jobs:
+  # Run tests.
+  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I think this doesn't really matters because it is the OS of the GitHub-hosted runner which runs the containers!

More info: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
So basically the only thing that matters is the runtime of the container itself. We can try to bump the Dockerfile runtime to 20.04 as soon as you're ready but we still have plenty of time to do this (18.04 is supported until 2023).